app/src/main/java/RetrofitClientInstance.kt

```diff
@@ -4,13 +4,13 @@ import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
-    private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://pokeapi.co/api/v2/pokemon/"
+    private var retrofit: Retrofit? = null
```

Comment:
Removed unnecessary semicolon since Kotlin doesn't require it at the end of the code.